sky/clouds/cloud_registry.py

```diff
@@ -15,6 +15,9 @@ class _CloudRegistry(dict):
     def from_str(self, name: Optional[str]) -> Optional['cloud.Cloud']:
         if name is None:
             return None
+        # map `k8s` to `kubernetes` as shortcut
+        if name == 'k8s':
```

Comment:
@romilbhardwaj @asaiacai We could also consider making `k8s` the main string and `kubernetes` the alias? This way the shorter string will be printed in messages like
```
» sky launch -t xxx                                                                                          
ValueError: Invalid instance type 'xxx' for any cloud among [AWS, Azure, GCP, IBM, Kubernetes, Lambda, Local, OCI, SCP].
```

Reply:
I'm ok with having the shorthand in the cloud name suggestion help strs (e.g., misspelled, not specified). 

That said, we should stick with the full name in error strs (e.g. "Kubernetes cluster does not have enough resources") and elsewhere in the code for readability.

Reply:
Swapped so now `k8s` is the main string and `kubernetes` is the alias.
example error message is now
```
(sky) $ sky launch --cloud xxx
ValueError: Cloud 'xxx' is not a valid cloud among ['aws', 'azure', 'gcp', 'ibm', 'k8s', 'lambda', 'local', 'oci', 'scp']
```